### PR TITLE
Fix SystemDB datasource from configuration properties

### PR DESCRIPTION
**Pull request by entrery**
Changes in the documentation to be merged with:
- https://github.com/dirigible-io/dirigible-io.github.io/pull/81

### CODE REVIEW

**Comment by entrery** [comment]
@delchev @vmutafov @ThuF please review this small fix

File: components/core/core-database/src/main/java/org/eclipse/dirigible/components/database/DataSourceSystemConfig.java [hunk 1]
```diff
@@ -44,13 +45,13 @@ public class DataSourceSystemConfig {
     public HikariDataSource getDataSource() {
         DataSourceProperties dataSourceProperties = new DataSourceProperties();
         dataSourceProperties.setName("SystemDB");
-        dataSourceProperties.setDriverClassName("org.h2.Driver");
-        dataSourceProperties.setUrl("jdbc:h2:file:./target/dirigible/h2/SystemDB");
-        dataSourceProperties.setUsername("sa");
-        dataSourceProperties.setPassword("");
+        dataSourceProperties.setDriverClassName(Configuration.get("DIRIGIBLE_DATASOURCE_SYSTEM_DRIVER", "org.h2.Driver"));
```
Comment (delchev) [inline components/core/core-database/src/main/java/org/eclipse/dirigible/components/database/DataSourceSystemConfig.java:48]: Let's use prefix DIRIGIBLE_DATABASE_*
and add documentation here:
https://www.dirigible.io/help/setup/setup-environment-variables/
and add definition here:
https://github.com/eclipse/dirigible/blob/abadf3a1259f5c4b4e08983d0913a735bb45644d/modules/commons/commons-config/src/main/java/org/eclipse/dirigible/commons/config/Configuration.java#L504

**Review by delchev — CHANGES_REQUESTED** [review]

File: components/core/core-database/src/main/java/org/eclipse/dirigible/components/database/DataSourceSystemConfig.java [hunk 2]
```diff
@@ -45,10 +45,10 @@ public class DataSourceSystemConfig {
...
         DataSourceProperties dataSourceProperties = new DataSourceProperties();
         dataSourceProperties.setName("SystemDB");
-        dataSourceProperties.setDriverClassName("org.h2.Driver");
-        dataSourceProperties.setUrl("jdbc:h2:file:./target/dirigible/h2/SystemDB");
-        dataSourceProperties.setUsername("sa");
-        dataSourceProperties.setPassword("");
+        dataSourceProperties.setDriverClassName(Configuration.get("DIRIGIBLE_DATABASE_SYSTEM_DRIVER", "org.h2.Driver"));
+        dataSourceProperties.setUrl(Configuration.get("DIRIGIBLE_DATABASE_SYSTEM_URL", "jdbc:h2:file:./target/dirigible/h2/SystemDB"));
+        dataSourceProperties.setUsername(Configuration.get("DIRIGIBLE_DATABASE_SYSTEM_USERNAME", "sa"));
+        dataSourceProperties.setPassword(Configuration.get("DIRIGIBLE_DATABASE_SYSTEM_PASSWORD", ""));
```
Comment (iliyan-velichkov) [inline components/core/core-database/src/main/java/org/eclipse/dirigible/components/database/DataSourceSystemConfig.java:51]: you can add the constants [here](https://github.com/eclipse/dirigible/blob/master/modules/commons/commons-config/src/main/java/org/eclipse/dirigible/commons/config/Configuration.java#L44) as well

**Review by iliyan-velichkov — APPROVED** [review]

**Review by delchev — APPROVED** [review]

**Comment by iliyan-velichkov** [comment]
@delchev note that for Default DS we use [DIRIGIBLE_DATASOURCE_DEFAULT.*](https://github.com/search?q=repo%3Aeclipse%2Fdirigible+DIRIGIBLE_DATASOURCE_DEFAULT&type=code)

**Pull request merged**